package.json

```diff
@@ -119,15 +119,12 @@
 		"yaml": "^1.10.2"
 	},
 	"devDependencies": {
-		"@emotion/react": "^11.7.1",
-		"@emotion/styled": "^11.6.0",
-		"@material-ui/core": "^4.12.3",
```

Comment:
@msivasubramaniaan As we have removed this, should the reference of this in files be also updated such as here: https://github.com/redhat-developer/vscode-openshift-tools/blob/main/src/webview/cluster/app/cluster.style.tsx#L6

We need to be update all the location where the references are added for old dependency.